background/test/app.test.js

```diff
@@ -43,38 +43,48 @@ describe('behavior of cloud function', function() {
     } catch (err) {
       console.log("Wasn't able to delete Google Cloud Functions");
     }
-    try {
-      cp.execSync(
-        `firebase firestore:delete --project ${projectId} -r translations`
-      );
-    } catch (err) {
-      console.log("Wasn't able to delete firestore project");
-    }
+    console.log('before deletion');
+    const db = new Firestore({
+      project: projectId,
+    });
+    await db
+      .collection('/translations')
+      .get()
+      .then(res => {
+        res.forEach(element => {
+          element.ref.delete();
+        });
+      });
+    console.log('after deletion');
   });
 
   it('should get the correct website', async () => {
-    return await request(app)
-      .get('/')
-      .retry(5)
-      .expect(200);
+    const body = await fetch(`${app}/`);
+    const res = await body.status;
+    console.log(res);
+    expect(res).to.equal(200);
   });
 
-  it('should ask for a translation', async () => {
-    return await request(app)
-      .post('/request-translation')
-      .type('form')
-      .send({lang: 'en', v: 'como estas'})
-      .retry(5)
-      .expect(200);
+  it('should get the correct response', async () => {
+    const params = new URLSearchParams();
+    params.append('lang', 'en');
+    params.append('v', 'como estas');
+
+    const body = await fetch(`${app}/request-translation`, {
+      method: 'POST',
+      body: params,
+    });
+    console.log(await body.text());
+    const res = await body.status;
+    expect(res).to.equal(200);
```

Comment:
I was a little surprised to see chai style assertions.  Is chai widely used in this repo?

Reply:
No; a lot of the assertions use supertest. However, since I'm using fetch instead of supertest, I wanted to add expect to confirm the statuses. What would you recommend instead?

Reply:
Through most of the cloud libraries tests, we use the `assert` module baked into node.js core:

```js
const assert = require('assert');

//....
assert.strictEqual(res, 200);
```

I like that it doesn't require a third party dependency, and is readable by everyone.